Third party libraries are located under /usr/local in FreeBSD (#477)
ansible/pylibssh: docs/changelog-fragments/477.bugfix.rst

> - Fix missing includes under FreeBSD -- by :user:`donnerhacke`

Reviewer: It's already dashed in Towncrier's Jinja2 template + it makes sense to use the past tense with more details here:
```suggestion
Extended CFLAGS to include header locations typical to FreeBSD
(:file:`/usr/local/include`) -- by :user:`donnerhacke`
```

Author: Thank you for the idea. Took it.

> Extended CFLAGS to include header locations typical to FreeBSD

Reviewer: ```suggestion
Extended ``CFLAGS`` to include header locations typical to FreeBSD
```

> (:file:/usr/local/include) -- by :user:`donnerhacke`

Reviewer: ```suggestion
(:file:`/usr/local/include`) -- by :user:`donnerhacke`
```

Reviewer: (missing such bits of syntax is exactly why it's best to accept the suggestions from GH UI)
